.github/workflows/check_help_text.yml

```diff
@@ -0,0 +1,55 @@
+name: Check Help Text
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  check-help-text:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install dependencies (Linux)
+        if: runner.os == 'Linux'
+        run: sudo apt install cmake ninja-build python3 build-essential gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib libusb-1.0-0-dev
```

Comment:
Do you really need all the cross-build tools just to compile `picotool`?

Reply:
Nope - I'd just copied it from the other workflow, which does need them as it sets USE_PRECOMPILED=false for some of the tests - will remove them now